docs/EmptyState.md

```diff
@@ -15,7 +15,7 @@ The `<EmptyState>` component is an element that can be used as a placeholder whe
 ```tsx
 import { EmptyState } from '@pxblue/react-components';
 
-<EmptyState icon={<AlertIcon style={{ fontSize: '100px', marginBottom: '15px' }} />} title={'No Alarms Found'} />;
+<EmptyState icon={<AlertIcon style={{ fontSize: '96px', marginBottom: '15px' }} />} title={'No Alarms Found'} />;
```

Comment:
Can we drop the style tag from the example? It shouldn't be required for someone to use it (if I'm not mistaken the component should handle the margin and sizing if we use inherit
```suggestion
<EmptyState icon={<AlertIcon fontSize: 'inherit' />} title={'No Alarms Found'} />;
```

Reply:
Does this suggestion actually work?

```
<EmptyState icon={<AlertIcon fontSize: 'inherit' />} title={'No Alarms Found'} />;
```

I thought we would need to wrap the fontSize in a style object.
```
<EmptyState icon={<AlertIcon style={{fontSize: 'inherit'}} />} title={'No Alarms Found'} />;
```